plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.html

```diff
@@ -258,10 +274,17 @@ <h2 id="Cybersecurity_PAB_Layer_Extensions">Cybersecurity PAB Layer Extensions</
 		<p>
 			<img border="0" src="Images/layers.png"/>
 		</p>
+		<h3 id="Assets_Layer">Assets Layer</h3>
+		<p>The Assets Layer will display in the diagram the inserted assets. The elements that are threatened by them will be highlighted with the same border color as the asset color. If two ore more assets threat the same element and at least two of them are displayed in diagram, the border color of the element will be black.</p>
+		<p>
+			<img border="0" src="Images/pab_diagram_4.png"/> 
+		</p>
+		<p>When the primary asset covers a functional chain, the involved functions and functional exchanges will be highlighted in the same way (if the primary asset is displayed).
+			If FunctionalChains (FC) are displayed in diagram, they will override the colors of Primary Asset Layer!</p>
 		<h3 id="Trust_Boundaries_Layer">Trust Boundaries Layer</h3>
-		<p>The Trust Boundaries Layer will change the diagrams appearence: Trusted components and actors will change their background color to white, untrusted components and actors will change their color to red. Communication edges between Trusted and Untrusted elements, so called Trust Boundaries, will be decorated with a small flame icon:</p>
+		<p>The Trust Boundaries Layer will change the diagrams appearance: Trusted components and actors will change their background color to white, untrusted components and actors will change their color to red. Communication edges between Trusted and Untrusted elements, so called Trust Boundaries, will be decorated with a small flame icon covered by a red border:</p>
 		<p>
-			<img border="0" src="Images/pab_diagram_2.png"/> 
+			<img border="0" src="Images/pab_diagram_2(new).png"/> 
 		</p>
```

Comment:
Add also in Assets Layer (and mention it in supporting assets layer), that 2 tools are available in this layer, under the Cybersecurity section (show/hide Primary Assets and Threats).